server/src/main/java/org/apache/cloudstack/backup/BackupManagerImpl.java

```diff
@@ -945,6 +985,9 @@ public List<Class<?>> getCommands() {
         cmdList.add(RestoreBackupCmd.class);
         cmdList.add(DeleteBackupCmd.class);
         cmdList.add(RestoreVolumeFromBackupAndAttachToVMCmd.class);
+        cmdList.add(AddBackupRepositoryCmd.class);
+        cmdList.add(DeleteBackupRepositoryCmd.class);
+        cmdList.add(ListBackupRepositoriesCmd.class);
```

Comment:
Update cmd required? to update name or mount options?

Reply:
@Pearl1594 maybe we can export/enable the APIs only when nas plugin is enabled?

Reply:
Actually I double checked, plugins are configureable even on a per zone level - so we'll need to export all the APIs when the feature is enabled.